### ISSUE TITLE
Tooltip is not defined for ‘Variable options’ button

### ISSUE THREAD

**Issue by MicrosoftIssueBot** [microsoft/calculator#1179]
**Repro Steps:** 
1. Launch "Calculator" app. 
2. Home screen will be displayed.
3. Navigate to menu button and activate it.
4. Navigate to "Graphing" list item and activate it.
5. Navigate to "Variable options" button.
6. Observe the issue.

**Actual Result:**
Tooltip is not defined for ‘Variable options’ button.

**Expected Result:**
Tooltip should be defined for ‘Variable options’ button.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I created this issue automatically as requested by a team member.
